README.md

```diff
@@ -87,7 +87,7 @@ Set the public key, private key and other required parameters. The `RavePayManag
 | onStagingEnv(boolean) | Set to `true` if you want your transactions to run in the staging environment otherwise set to `false`. Defaults to false  | `boolean` | Not Required
 | allowSaveCardFeature(boolean) | Set to `true` if you want your users to be able to save their cards for later reuse otherwise set to `false` | `boolean` | Required
 | setMeta(`List<Meta>`) | Pass in any other custom data you wish to pass. It takes in a `List` of `Meta` objects | List<Meta> | Not Required
-|setSubAccounts(`List<SubAccount>`) | Pass in a `List` of `SubAccount`,if you want to split transaction fee with other people | `List<SubAccount>` | Not Required
+|setSubAccounts(`List<SubAccount>`) | Pass in a `List` of `SubAccount`,if you want to split transaction fee with other people. Subaccounts are your vendors' accounts that you want to settle per transaction. You can look up more on Subaccounts and how to create them [here](https://developer.flutterwave.com/docs/split-payment) | `List<SubAccount>` | Not Required
```

Comment:
@Bolaji95 add an example of how to create the SubAccount object. Also, add a link to "learn more about split payments and subaccounts" that leads to the page on the documentation

Reply:
@biodunalfet Done.